crates/top/re_sdk/src/web_viewer.rs

```diff
@@ -120,6 +120,7 @@ pub fn host_web_viewer(
     bind_ip: &str,
     web_port: WebViewerServerPort,
     force_wgpu_backend: Option<String>,
+    video_decoder: Option<String>,
```

Comment:
Can this use `DecodeHardwareAcceleration` to avoid stringly types?

Reply:
no can't because we don't have the native viewer (and thus no re_renderer) here